manage-data/data-store/index-basics.md

```diff
@@ -85,7 +83,7 @@ If you use {{es}} {{security-features}}, the following [security privileges](ela
 * The `view_index_metadata` and `manage` index privileges to view a data stream or index’s data.
 * The `manage_index_templates` cluster privilege to manage index templates.
 
-To add these privileges, go to **Stack Management > Security > Roles** or use the [Create or update roles API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-security-put-role).
+To add these privileges, go to **Security > Roles** or use the [Create or update roles API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-security-put-role).
```

Comment:
this step doesn't work without `stack management`

Reply:
Fixed in [c643263](https://github.com/elastic/docs-content/pull/3383/commits/c643263b874de0e0929253c37bbcebec294148eb)